base/src/main/java/io/spine/protobuf/Durations2.java

```diff
@@ -59,6 +60,7 @@
 public final class Durations2 {
 
     public static final Duration ZERO = fromMillis(0L);
+    static final String PROTOBUF_DURATIONS = "com.google.protobuf.util.Durations";
```

Comment:
Please get the canonical name from the class literal.

Reply:
The compiler says `Attribute value must be a constant`.